tests/foreman/api/test_template_combination.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+"""Tests for template combination
+
+@Requirement: TemplateCombination
+
+@CaseAutomation: Automated
+
+@CaseLevel: Acceptance
+
+@CaseComponent: API
+
+@TestType: Functional
+
+@CaseImportance: Medium
+
+@Upstream: No
+"""
+from nailgun import entities
+
+from robottelo.decorators import tier1, skip_if_bug_open
+from robottelo.test import APITestCase
+
+
+def _delete_entities(*entities):
+    for ent in entities:
+        ent.delete()
+
+
+class TemplateCombinationTestCase(APITestCase):
+    """Implements TemplateCombination tests"""
+
+    @classmethod
+    def setUpClass(cls):
+        super(TemplateCombinationTestCase, cls).setUpClass()
```

Comment:
Please add a docstring for `setUpClass()`, `tearDownClass()`, `setUp()` and `tearDown()` methods.